docs/src/operate/troubleshooting/tedge-diag.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Collecting diagnostic logs
+tags: [Operate]
+sidebar_position: 1
+description: How to collect logs and system diagnostics using plugin-based automation
+---
+
+%%te%% helps you collect all diagnostic information about your device and services with a single command running user-defined diagnostic plugins.
```

Comment:
```suggestion
%%te%% diagnostic collector helps you collect all diagnostic information about your device and services with a single command which can be further extended with user-defined diagnostic plugins.
```
As this tool is not just about user-defined plugins but also the inbuilt plugins.

Reply:
Applied all your suggestions here 81c06445287266e448203d1b881a9ee1fedcb464.